gen/top_gen.ml

```diff
@@ -422,6 +422,10 @@ let max_set = IntSet.max_elt
             vs in
         build_observers p i x vs in
 
+    (* string ->
```

Comment:
whats the purpose of this comment and the todo after?

Reply:
Both these comments and the TODO are no longer used in this function. I will remove.